packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx

```diff
@@ -34,6 +34,14 @@ export type ChatCompositeProps = {
  * Additional customizations for the chat composite
  */
 export type ChatOptions = {
+  /**
+   * UNSTABLE: Feature flag to enable ErrorBar.
+   *
+   * This option will be removed once ErrorBar is stable.
```

Comment:
IMO even when the error bar becomes stable, it's good to have it as an optional. CONTOSO might not want to show the error bar...

Reply:
Could also tag it `@experimental` https://tsdoc.org/pages/tags/experimental/

Reply:
Tagged it @experimental.

Not sure we want to keep this option later on, so playing it safe.